src/Commands/Runner.h

```diff
@@ -595,8 +656,9 @@ class Runner {
         std::map<std::string, Command*>::iterator itCommands =
             _commands.find(topic);
         if (itCommands == _commands.end()) {
-          _output << "\"" << topic << "\" is not a valid command"
-                                      " name.\n";
+          _output << "\"" << topic
```

Comment:
> I think we should take a hard dependency instead of building / running / documenting two "modes". I talked with Tim about this and he was ok with a hard dependency. This change would also need to update README.md and CONTRIBUTING.md to account for this.

I am perfectly fine with taking hard dependency. I actually prefer it but did not want to make such change without hearing from you and Tim. I am thinking of adding replxx as third-party submodule so we can have exact control of the version we use and always use static linking.


> wrapping seems off here. What if you merge the final string literal sequence?

I leave all wrapping to clang-format. I will try merging the final string literal

Reply:
A submodule sounds good to me.